content/client-server-api/modules/push.md

```diff
@@ -155,8 +155,12 @@ The different `kind`s of rule, in the order that they are checked, are:
 1.  **Content-specific rules (`content`).**
     These configure behaviour for (unencrypted) messages that match certain
```

Comment:
I wonder if we should remove this "(unencrypted)" in this section -- I think this isn't really true since clients also run push rules? (Although see #1401)

Reply:
Indeed.

Reply:
Is that a "Yes, remove this please"? 😄

Reply:
There seems to be a few more instances of it, I think it'll be cleaner to do it in a follow-up (some of them are quite unrelated to this PR).

Reply:
I started doing this in #1461.